easybuild/easyconfigs/i/intel/intel-2019a.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'Toolchain'
+
+name = 'intel'
+version = '2019a'
+
+homepage = 'http://software.intel.com/en-us/intel-cluster-toolkit-compiler/'
```

Comment:
Wonder if we should change this to https://easybuild.readthedocs.io/en/develop/Common-toolchains.html#intel-toolchain